lib/grit_adapter/git_layer_grit.rb

```diff
@@ -225,6 +227,10 @@ def refs(options, prefix)
       def repo
         @repo ||= Grit::Repo.new(@git.git_dir)
       end
+
+      def apply_patch(ref, patch=nil)
```

Comment:
I know you've just copied this method signature, but I'm wondering what the result would be of `apply_patch` when `patch=nil`. Any ideas? Is this method ever called with just a ref?